caluma/caluma_form/api.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Optional
+
+from caluma.caluma_form import domain_logic, models
+from caluma.caluma_user.models import BaseUser
+
+
+def save_answer(
+    question: models.Question,
+    document: models.Document,
+    user: BaseUser,
```

Comment:
I suggest typing `user: Optional[User]` instead, which would make usage from scripts a bit easier

Reply:
What would be the default here, if no `user` is provided? `AnonymousUser`?
The goal would be that `created_by_user` and `created_by_group` are populated too, so we need a default.

Reply:
The default would be `None` for both fields, which in case of script-generated data would be the right thing anyway (and it's also why the `created_by_` fields in the model are explicitly nullable

Reply:
Ah I see, valid point :)